app/services/questions_service.rb

```diff
@@ -1,4 +1,6 @@
 class QuestionsService
+  DA_CODE_REGEXP = /DA\d{3}/
```

Comment:
Could we use `DOMESTIC_ABUSE_` instead of `DA_` to make it clearer what the constant refers to?